minefana-common/src/main/java/com/github/games647/minefana/common/InfluxConnector.java

```diff
@@ -35,7 +35,9 @@ protected void init() {
 
         // Flush every 2000 Points, at least every 1s
         influxDB.enableBatch(2_000, 2, TimeUnit.MINUTES);
-        influxDB.enableBatch(BatchOptions.DEFAULTS.jitterDuration(500));
+        //influxDB.enableBatch(BatchOptions.DEFAULTS.jitterDuration(500));
```

Comment:
What's the problem with that? If that causes issues please add a comment line too.